.github/workflows/arm64-build-checks.yml

```diff
@@ -0,0 +1,46 @@
+name: arm64 build checks
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependency package
+      run: |
+        sudo sed -i -E 's|^deb ([^ ]+) (.*)$|deb [arch=amd64] \1 \2\ndeb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports/ \2|' /etc/apt/sources.list
+        sudo dpkg --add-architecture arm64
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends \
+                crossbuild-essential-arm64 \
+                git \
+                cmake \
+                ca-certificates \
+                libpython-dev:arm64 \
+                libpython3-dev:arm64 \
+                python-numpy \
+                python3-numpy \
```

Comment:
I try to install `libpython-dev:arm64 libpython3-dev:arm64 python-numpy:arm64 python3-numpy:arm64` first,
```
-DENABLE_PYTHON=ON \
-DPYTHON_EXECUTABLE=/usr/bin/python3.6 \
-DPYTHON_INCLUDE_DIR=/usr/include/python3.6 \
-DPYTHON_LIBRARY=/usr/lib/aarch64-linux-gnu/libpython3.6m.so
```
And I try to build with above extra options, but looks like the python2/3 binding is still unavailable, did I miss something?

Reply:
These are not OpenCV's flags. Use
```
        -DPYTHON3_INCLUDE_PATH="/usr/include/python3.6" \
        -DPYTHON3_NUMPY_INCLUDE_DIRS="/usr/local/lib/python3.6/dist-packages/numpy/core/include" \
        -DPYTHON3_CVPY_SUFFIX=".cpython-37m-aarch64-linux-gnu.so" \
```
(make sure you have installed `libpython3-dev:armhf` and `python3-numpy`)

Reply:
@dkurt Thanks! it works:
```
apt-get install -y libpython-dev:arm64 libpython3-dev:arm64 python-numpy:arm64 python3-numpy:arm64
cmake -DPYTHON2_INCLUDE_PATH=/usr/include/python2.7/ \
      -DPYTHON2_NUMPY_INCLUDE_DIRS=/usr/lib/python2.7/dist-packages/numpy/core/include \
      -DPYTHON3_INCLUDE_PATH=/usr/include/python3.6m/ \
      -DPYTHON3_CVPY_SUFFIX=.cpython-36m-aarch64-linux-gnu.so \
      -DPYTHON3_NUMPY_INCLUDE_DIRS=/usr/lib/python3/dist-packages/numpy/core/include \
      -DCMAKE_TOOLCHAIN_FILE=../platforms/linux/aarch64-gnu.toolchain.cmake \
      -DOPENCV_EXTRA_MODULES_PATH=../../opencv_contrib/modules ../
```
and python info as below:
```
--   Python 2:
--     Interpreter:                 /usr/bin/python2.7 (ver 2.7.17)
--     Libraries:
--     numpy:                       /usr/lib/python2.7/dist-packages/numpy/core/include (ver undefined - cannot be probed because of the cross-compilation)
--     install path:                lib/python2.7/dist-packages/cv2/python-2.7
--
--   Python 3:
--     Interpreter:                 /usr/bin/python3 (ver 3.6.9)
--     Libraries:
--     numpy:                       /usr/lib/python3/dist-packages/numpy/core/include (ver undefined - cannot be probed because of the cross-compilation)
--     install path:                lib/python3.6/dist-packages/cv2/python-3.6
--
--   Python (for build):            /usr/bin/python2.7
```
Looks like we still need the `PYTHON2_LIBRARIES=/usr/lib/aarch64-linux-gnu/libpython2.7.so` and `PYTHON3_LIBRARIES=/usr/lib/aarch64-linux-gnu/libpython3.6m.so`, right?

Reply:
Cannot say exactly. Can you try to build and check?

Reply:
```
--     To be built:                 aruco bgsegm bioinspired calib3d ccalib core datasets dnn dnn_objdetect dnn_superres dpm face features2d flann fuzzy gapi hfs highgui img_hash imgcodecs imgproc intensity_transform line_descriptor mcc ml objdetect optflow phase_unwrapping photo plot python2 python3 quality rapid reg rgbd saliency shape stereo stitching structured_light superres surface_matching text tracking ts video videoio videostab xfeatures2d ximgproc xobjdetect xphoto
--     Disabled:                    world
--     Disabled by dependency:      -
--     Unavailable:                 alphamat cnn_3dobj cudaarithm cudabgsegm cudacodec cudafeatures2d cudafilters cudaimgproc cudalegacy cudaobjdetect cudaoptflow cudastereo cudawarping cudev cvv freetype hdf java js julia matlab ovis sfm viz
```

the "to be built" is same as above(with lib flag or without lib flag), and py2 and py3 is included.